### PR TITLE
[feat]: Added Cart Component

### PR DESCRIPTION
**Pull request by raghav2110**
Hi , I have added cart component for my ecom please review it.

### CODE REVIEW

File: src/components/PriceSummary.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+export function PriceSummary({ products }){
+    console.log(products)
```
Comment (Hamdrive) [inline src/components/PriceSummary.jsx]: Take care of cleaning up consolelogs

File: src/styles/wishList.css [hunk 2]
```diff
@@ -31,7 +31,17 @@
 
 .btn-addToCart{
     margin: 0.625rem;
-    background-color: #FFA500;
+    background-color: #F7CA00;
```
Comment (Hamdrive) [inline src/styles/wishList.css:34]: If the styles are being used in more than one place, consider defining it as global styles instead

**Review by Hamdrive — COMMENTED** [review]
Liked the implementation of context for cart and wishlist
Will suggest to take care of repetitive styles and keep them global
Overall good work done,

File: src/components/PriceSummary.jsx (hunk 1 above)
Comment (raghav2110) [reply src/components/PriceSummary.jsx]: Yes will take care of this. Thanks for poinitng out.

**Review by hinanshisuthar — COMMENTED** [review]
- Good code structure
- Suggest to clean up consoles
- Functionalities work as intended
- Overall logic implementation is nice
- Ready to merge

**Pull request merged**